server/server.go

```diff
@@ -47,8 +48,13 @@ func ListenAndServe(ctx context.Context, httpsPort, httpPort int, handler http.H
 		opts.TLSListenerConfig.TLSConfig = &tls.Config{}
 	}
 
+	logLevel := logrus.InfoLevel
+	// Use the same env-vars as webhook to log at the debug level
+	if os.Getenv("CATTLE_DEBUG") == "true" || os.Getenv("RANCHER_DEBUG") == "true" {
+		logLevel = logrus.DebugLevel
+	}
 	logger := logrus.StandardLogger()
-	errorLog := log.New(logger.WriterLevel(logrus.DebugLevel), "", log.LstdFlags)
+	errorLog := log.New(logger.WriterLevel(logLevel), "", log.LstdFlags)
```

Comment:
I'm trying to get the following to run to verify the behavior, but nothing is printed no matter what log level is set:

```go
package main

import (
	"log"
	"os"

	"github.com/sirupsen/logrus"
)

func main() {
	logLevel := logrus.ErrorLevel
	// Use the same env-vars as webhook to log at the debug level
	if os.Getenv("CATTLE_DEBUG") == "true" || os.Getenv("RANCHER_DEBUG") == "true" {
		logLevel = logrus.DebugLevel
	}
	logger := logrus.StandardLogger()
	errorLog := log.New(logger.WriterLevel(logLevel), "", log.LstdFlags)
	errorLog.Println("HI")
}
```

Does that run for you?